modules/common/src/main/scala/protocol.scala

```diff
@@ -32,7 +32,12 @@ sealed trait SerializationType extends Product with Serializable
 case object Protobuf           extends SerializationType
 case object Avro               extends SerializationType
 
-class rpc(val serializationType: SerializationType)                   extends StaticAnnotation
+sealed abstract class CompressionType extends Product with Serializable
+case object Identity                  extends CompressionType
```

Comment:
I'm wondering if we could rename this by something like as `Plain` or similar, what do you think?

Reply:
Yeah, you know, I was trying to be cohesive with [grpc-java](https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/Codec.java) but I agree it's not the best name.

Reply:
I missed that, forget my suggestion ;)